### ISSUE TITLE
TimeSeriesSVR not working 

### ISSUE THREAD

**Issue by tapiwa7** [rtavenar/tslearn#64]
i am having a regression problem. i have 2 datasets, a training(Z) and testing dataset(Test1 ). The target is y. i have 6 months of data (counts). When i predict the test dataset it gives me back the same  number for every row.What am i doing wrong? Below is the code and results
print(y.shape)
Z = to_time_series_dataset(Z)
Test1 =  Test1.values
Test1 = to_time_series_dataset(Test1)
print(Z.shape)
print(Test1.shape)

reg = TimeSeriesSVR(sz = 6,d =1 , kernel="gak", gamma="auto")
model=reg.fit(Z, y)
pred =model.predict(Test1)
print(pred)

output
(17,)
(17, 6, 1)
(162, 6, 1)
[0.33529411 0.33529411 0.33529411 0.33529411 0.33529411 0.3352941
 0.33529411 0.33529411 0.33529411 0.33529411 0.33529411 0.33529411
 0.33529411 0.33529408 0.33529411 0.33529387 0.33529412 0.33529411
 0.33529411 0.33529411 0.33529411 0.33529411 0.33529411 0.33529411
 0.33533446 0.33529411 0.33529411 0.33529411 0.33529411 0.33529411
 0.33509248 0.33529409 0.33529411 0.33529411 0.33525814 0.33529411
 0.33529411 0.33529411 0.33529411 0.33529411 0.33529411 0.33529411

Its showing same result for all 162 rows

**Comment by GillesVandewiele** [comment]
Could you check the output when you call `model.predict(Z)`? Does it give constant predictions as well? If so, the model probably didn't learn too much and is just predicting the class distributions of the training set. If not, then maybe you normalized your training data but forgot to normalize the test data?

**Comment by tapiwa7** [comment]
okay this was fixed. it doesnt show constant predictions anymore but the accuracy is off. Totally. Any ideas on how to fix this?

**Comment by GillesVandewiele** [comment]
So what did you change in order to fix it? Are you sure your labels are correct and similar in test and train-set? You could maybe print set(predictions), set(y_train) and set(y_test), these should all three be equal...

**Comment by tapiwa7** [comment]
I just added more rows as it was difficult for the model to work on . Below are the shapes of train and test data. I dont think its the data thats a problem. 
(60000, 6, 1)
(20000, 6, 1)

**Comment by GillesVandewiele** [comment]
The shapes of the data are alright, but what are the unique labels in the train and test set?

I'm guessing (which is all I can do given the information provided) that for example the labels in your train set (or the labels of your predictions) are 0 and 1 while they are 1 and 2 in your test set

**Comment by tapiwa7** [comment]
so what i am doing is in the training i am putting both the data and labels. In the test i am only feeding in the data and predicting the labels. When i then compare actuals with predicted its totally off

**Comment by GillesVandewiele** [comment]
and so set(Test1) is EQUAL to set(pred) in your code snippet? I'm out of inspiration then of possible mistakes :), it's only 6 variables so maybe plot some distribution plots of training and test data and check if there are any large differences?

**Comment by rtavenar** [comment]
Hi @tapiwa7 ,

As @GillesVandewiele suggested, if you could provide us with a minimum working example, that would help a lot, otherwise it is really difficult to guess whether the issue is in `tslearn` code or in your usage of the library.
